Upstream-Status: Pending

diff -aur a/gcc/configure b/gcc/configure
--- a/gcc/configure	2025-04-07 18:47:48.268714622 +0300
+++ b/gcc/configure	2025-04-07 18:51:40.669642133 +0300
@@ -15145,7 +15145,7 @@
     */t-linux64 )
        if test -f ./config/$f
        then
-         tmake_file_="${tmake_file_} ./config/$f"
+         tmake_file_="${tmake_file_} \$(or \$(GCC_BUILD_DIR), .)/config/$f"
        fi
        ;;
     * )
diff -aur a/gcc/configure.ac b/gcc/configure.ac
--- a/gcc/configure.ac	2025-04-07 18:47:48.436721123 +0300
+++ b/gcc/configure.ac	2025-04-07 18:51:33.276418349 +0300
@@ -2374,7 +2374,7 @@
     */t-linux64 )
        if test -f ./config/$f
        then
-         tmake_file_="${tmake_file_} ./config/$f"
+         tmake_file_="${tmake_file_} \$(or \$(GCC_BUILD_DIR), .)/config/$f"
        fi
        ;;
     * )
diff -aur a/libada/Makefile.in b/libada/Makefile.in
--- a/libada/Makefile.in	2024-08-01 11:17:17.000000000 +0300
+++ b/libada/Makefile.in	2025-03-29 18:44:33.590357546 +0200
@@ -42,6 +42,7 @@
 toolexeclibdir = @toolexeclibdir@
 enable_shared = @enable_shared@
 
+CC = @CC@
 LN_S=@LN_S@
 AWK=@AWK@
 
