Upstream-Status: Pending

diff -Naur a/gcc/configure a/gcc/configure
--- a/gcc/configure	2025-04-06 10:45:52.482821737 +0300
+++ a/gcc/configure	2025-04-06 10:46:59.403913913 +0300
@@ -15142,7 +15142,7 @@
     */t-linux64 )
        if test -f ./config/$f
        then
-         tmake_file_="${tmake_file_} ./config/$f"
+         tmake_file_="${tmake_file_} $PWD/config/$f"
        fi
        ;;
     * )
diff -Naur a/gcc/configure.ac a/gcc/configure.ac
--- a/gcc/configure.ac	2025-04-06 10:45:52.485821741 +0300
+++ a/gcc/configure.ac	2025-04-06 10:46:43.262891691 +0300
@@ -2374,7 +2374,7 @@
     */t-linux64 )
        if test -f ./config/$f
        then
-         tmake_file_="${tmake_file_} ./config/$f"
+         tmake_file_="${tmake_file_} $PWD/config/$f"
        fi
        ;;
     * )
diff -aur a/libada/Makefile.in b/libada/Makefile.in
--- a/libada/Makefile.in	2024-08-01 11:17:17.000000000 +0300
+++ b/libada/Makefile.in	2025-03-29 18:44:33.590357546 +0200
@@ -42,6 +42,7 @@
 toolexeclibdir = @toolexeclibdir@
 enable_shared = @enable_shared@
 
+CC = @CC@
 LN_S=@LN_S@
 AWK=@AWK@
 
